### PR TITLE
Pretty print json for kumactl pkg

### LINKED ISSUE #357: `kumactl` commands with `-ojson` option should prettify output JSON
### Summary

* `kumactl` commands with `-ojson` option should prettify output JSON, i.e.
  * `kumactl get meshes -ojson`
  * `kumactl inspect dataplanes -ojson`

### Steps To Reproduce

1. Run
```
$ kumactl get meshes -ojson
```

The output will be:
```
{"items":[{"mtls":{"ca":{"builtin":{}},"enabled":true},"name":"default","type":"Mesh"}]}
```

### Expected behaviour

* output JSON should be prettified, i.e.
```
{
  "items": [
    {
      "mtls": {
        "ca": {
          "builtin": {}
        },
        "enabled": true
      },
      "name": "default",
      "type": "Mesh"
    }
  ]
}
```

### PR DESCRIPTION
**Pull request by sterchelen**
### Summary

Pretty print json output for `kumactl` command

### Full changelog

* Implement pretty print thanks to `MarshalIndent`

### Issues resolved

Fix #357

### CODE REVIEW

File: app/kumactl/pkg/output/json/printer.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ var _ output.Printer = &printer{}
 type printer struct{}
 
 func (p *printer) Print(obj interface{}, out io.Writer) error {
-	b, err := json.Marshal(obj)
+	b, err := json.MarshalIndent(obj, "", "  ")
 	if err != nil {
 		return err
 	}
```
Comment (yskopets) [inline app/kumactl/pkg/output/json/printer.go]: Let's add a newline to the end (for consistency with `YAML` and `TABLE` output)

Comment (sterchelen) [reply app/kumactl/pkg/output/json/printer.go]: What do you mean by add a newline to the end ?
Tell me if I'm wrong, you want a newline to the end of the prettyprint output ? Am I right ?

Comment (yskopets) [reply app/kumactl/pkg/output/json/printer.go]: Yes, JSON should be followed by a newline.

E.g., 

```
b, err := json.MarshalIndent(obj, "", "  ")
if err != nil {
	return err
}
if _, err := out.Write(b); err != nil {
	return err
}
_, err = fmt.Fprintln(out)
return err
```

**Comment by yskopets** [comment]
Unit tests for this PR are part of #361

**Pull request merged**